### ISSUE TITLE
Inconsistency in config of sway/workspaces module

### ISSUE THREAD

**Issue by choreutes** [Alexays/Waybar#493]
I have come across the following issue while trying to configure the sway/workspaces module.

If I set `"format": "{name}"` for the module, the output in waybar is the configured icon from `"format-icons"`:
![Screenshot1](https://user-images.githubusercontent.com/1265189/67853897-b3bb7980-fb0f-11e9-9a5c-60eb37d24dd3.png)

If I set `"format": "{icon}"`, however, I get the name of the workspace configured in sway:
![Screenshot2](https://user-images.githubusercontent.com/1265189/67853898-b3bb7980-fb0f-11e9-9924-c96fac8b1b27.png)

The third option, `"format": "{index}"` works as expected. This seems like a mixup of the two options, but maybe I got something wrong. 

It also seems like there is some unintended whitespace to the left of the icon in the second case. This is most visible if I select the corresponding workspace because of the blue underline:
![Screenshot3](https://user-images.githubusercontent.com/1265189/67853899-b3bb7980-fb0f-11e9-8a30-29dbfc32d322.png)

Is there anything I have overlooked? Or are there more people experiencing the same thing?

**Comment by piwonskp** [comment]
I also have experienced the same behavior. In fact it is not the icon configured in format-icons (at least in my case - I have no icons configured in waybar config). `"format": "{name}"` does the same as strip_workspace_numbers in swaybar. I greatly appreciate the feature, but it seems undocumented. [https://github.com/Alexays/Waybar/wiki/Module:-Sway#config-2]() states that `{name}` is the default for format which appears not to be true.

My workspaces:
```
    set $workspace1 ""
    set $workspace2 ""
    set $workspace3 ""
    set $workspace4 ""
    set $workspace10 "10"
    bindsym $mod+1 workspace number 1: $workspace1
    bindsym $mod+2 workspace number 2: $workspace2
    bindsym $mod+3 workspace number 3: $workspace3
    bindsym $mod+4 workspace number 4: $workspace4
    bindsym $mod+0 workspace number 10: $workspace10
```
With format set as `{name}`:
![image](https://user-images.githubusercontent.com/11379770/71295617-07627a80-237d-11ea-8a6e-3b8203964efa.png)
Without format:
![image](https://user-images.githubusercontent.com/11379770/71295666-3842af80-237d-11ea-8fa0-d3daeea0d747.png)

**Comment by Alexays** [comment]
Name use the defined one in the sway config, that's why you got the icon if you defined the icon in that config :/
May we can rename the param `name` to `value` ?

**Comment by Alexays** [comment]
Fixed with 27fbea2b5afde7fd601aa4e2afb2b3a74c9110bf
Default format is now `value` and `name` is now correctly documented
